debian/rules

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/make -f
 
-# export DH_VERBOSE=1
-
-DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+#export DH_VERBOSE=1
 
 %:
-	dh_update_autotools_config
-	dh $@ --parallel --with autoreconf
```

Comment:
`--parallel` and `--with autoreconf` are the defaults as of `debhelper` compatibility level 10. As per `debhelper(7)`'s manual page:

```
v10
   - The autoreconf sequence is now enabled by default.
   - Debhelper will default to --parallel for all buildsystems that support parallel building.
```